app/assets/javascripts/application.js

```diff
@@ -13,4 +13,73 @@
 //= require jquery
 //= require jquery_ujs
 //= require turbolinks
+//= require bootstrap
+//= require interact/interact
 //= require_tree .
+
+interact('.resize-drag')
+    .draggable({
+        onmove: window.dragMoveListener
+    })
+    .resizable({
+        preserveAspectRatio: true,
+        edges: { left: true, right: true, bottom: true, top: true }
+    })
+    .on('resizemove', function (event) {
+        var target = event.target,
+            x = (parseFloat(target.getAttribute('data-x')) || 0),
+            y = (parseFloat(target.getAttribute('data-y')) || 0);
+
+        // update the element's style
```

Comment:
don't leave the comments in. Just pull those lines into a new, well-named function